project/api/migrations/0010_auto_20170110_1830.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from django.db import migrations, models
+from django.db import migrations
```

Comment:
Rather than touching migrations files, which are auto-generated by Django, lets just [add an ignore rule for migrations in our Flake8 configuration](http://flake8.pycqa.org/en/latest/user/configuration.html).

Reply:
Thanks for the information. I was unaware these files are auto generated.  I made the change in commit  https://github.com/CiviWiki/OpenCiviWiki/pull/69/commits/ef3f3454c2e9d42243dd26fb1f60764b1d132715